mappings/net/minecraft/loot/condition/TableBonusLootCondition.mapping

```diff
@@ -1,11 +1,11 @@
 CLASS net/minecraft/class_182 net/minecraft/loot/condition/TableBonusLootCondition
-	FIELD field_1234 enchantment Lnet/minecraft/class_1887;
+	FIELD field_1234 type Lnet/minecraft/class_1887;
```

Comment:
This field is called `enchantment` in the json format. https://minecraft.gamepedia.com/Loot_table#Functions